server/kolide/osquery.go

```diff
@@ -26,6 +26,15 @@ type OsqueryService interface {
 // osquery distributed query.
 type OsqueryDistributedQueryResults map[string][]map[string]string
 
+// OsqueryStatus represents osquery status codes (0 = success, nonzero =
+// failure)
+type OsqueryStatus int
+
+const (
+	// StatusOK is the success code returned by osquery
+	StatusOK OsqueryStatus = 0
```

Comment:
` StatusOK OsqueryStatus = iota` ?

Reply:
I'd prefer to leave it as the explicit zero since this isn't really an enum but a constant for the `0` status code returned from osquery.